kotlin-generator/src/main/scala/models/generator/kotlin/KotlinGenerator.scala

```diff
@@ -33,6 +33,7 @@ class KotlinGenerator
   class GeneratorHelper(service: Service) {
 
     private val nameSpace = makeNameSpace(service.namespace)
+    private val sharedNameSpace = "com.apicollective.shared"
```

Comment:
I think this should be `io.apibuilder.api` - that's where all the apibuilder code lives, for example `https://app.apibuilder.io/apicollective/apibuilder-api/0.14.13/original`

Reply:
`io.apibuilder.app` you mean?